.ebextensions/nginx.config

```diff
@@ -368,22 +372,14 @@ files:
             return 301 https://$host$request_uri;
           }
 
-          # Expire rules for static content
-
-          location ~* \.(?:css|js)$ {
+          location ~* \.(?:manifest|appcache|xml|json|pdf|docx?)$ {
             access_log off;
-            add_header Cache-Control "max-age=31536000";
+            add_header Cache-Control "max-age=0";
```

Comment:
Are you intentionally making this non-cacheable?  If so, you probably want to do `Cache-Control: no-cache, no-store, must-revalidate`.  Though, I'm not sure why you would want to.

Reply:
@hlieberman-gov For file types that match `manifest|appcache|xml|json|pdf|docx?`, yeah, we don't want to cache them because…

1. `/browserconfig.xml` will change if the assets it references change,
2. same deal with `/manifest.json`, and
3. we're serving some PDFs and DOCX files straight from `/documents` (`/public/documents` in the app) which won't have hashed file names.

I _think_ that makes sense. What do you think? If so, I'll make the update to use `no-cache, no-store, must-revalidate`.